mgmt/rest/client/client.go

```diff
@@ -104,9 +105,11 @@ func Username(u string) metaOp {
 var (
 	secureTransport = &http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: false},
+		IdleConnTimeout: time.Second,
```

Comment:
Not really. What scenario do you see where it would be beneficial for a user to configure this?